src/consensus/ledger_enclave.h

```diff
@@ -53,7 +62,9 @@ namespace consensus
       const uint8_t* data,
       size_t size,
       bool globally_committable,
-      bool force_chunk = false)
+      bool force_chunk,
+      kv::Term term,
+      kv::Version index)
```

Comment:
Are these expected to be passed to the host?

Reply:
No. They could be, but I don't think we have a purpose for them, and they're not needed at the moment. The argument is that Raft _knows_ the term and index it is appending; logically it is calling `ledger.append(term, index, data)`, so it should pass those fields in case they're important to the receiver. For the stub ledger used in the unit tests, this is important. For this real implementation it's not (currently). But maybe at some point in the future, it would make sense to do the GcmHeader construction here (still within enclave, but logically part of the ledger representation rather than the 'raw' body)?

Reply:
I didn't see that they were used in the unit test logic, makes sense!